rdtools/plotting.py

```diff
@@ -230,3 +231,39 @@ def soiling_rate_histogram(soiling_info, bins=None):
     ax.set_ylabel('Count')
 
     return fig
+
+
+def availability_cumulative_plot(outage_info, cumulative_energy,
+                                 expected_energy):
+    """
+    Visualize detected outages and expected energy confidence intervals.
+
+    Parameters
+    ----------
+    outage_info : pd.DataFrame
+        The output dataframe of :py:func:`.availability.loss_from_energy`
+    cumulative_energy : pd.Series
+        The system's cumulative production timeseries
+    expected_energy : pd.Series
+        The system's expected energy timeseries
+
+    Returns
+    -------
+    fig: matplotlib Figure
+    """
+    fig, ax = plt.subplots()
+    cumulative_energy.plot(ax=ax, label='Cumulative Energy')
+    for i, row in outage_info.iterrows():
+        start, end = row[['start', 'end']]
+        start_energy = row['start_energy']
+        expected_production = row['expected_energy']
+        lo, hi = np.abs(expected_production - row[['ci_lower', 'ci_upper']])
+        raw_expected_energy = expected_energy.loc[start:end].sum()
+        scale_factor = expected_production / raw_expected_energy
```

Comment:
I think this scaling is bit different than the scaling calculation in `loss_from_energy()` would it make sense to align them? If so, perhaps make the scaling function stand alone so it can be reused. If you go with an OO approach, it could also be got from the object.